<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>midw</class>
 <widget class="QWidget" name="midw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>832</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 247, 247);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>60</y>
     <width>311</width>
     <height>771</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>771</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>310</width>
     <height>770</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="chatListW">
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>560</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="ListWidget" name="chatList">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>700</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="blankpage"/>
  </widget>
  <widget class="QLineEdit" name="searchlineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(218, 217, 215);
border-radius:20px;
</string>
   </property>
   <property name="placeholderText">
    <string>             搜索...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/res/pic/ic_add_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
